test/test_multiperiod_ac_powerflow.jl

```diff
@@ -119,3 +119,60 @@ end
     solve_powerflow!(data_brute_force; pf = pf)
     @test isnothing(data_brute_force.loss_factors)
 end
+
+@testset failfast = true "MULTI-PERIOD power flows evaluation with DS" for mode in (
+    :nothing,
+    :equal,
+    :dict,
+    :array_1,
+    :array_24,
+)
+    # get system
+    sys = PSB.build_system(PSB.PSITestSystems, "c_sys14"; add_forecasts = false)
+    generators = get_components(ThermalStandard, sys)
+    create_gspf(generators) =
+        Dict((ThermalStandard, get_name(x)) => abs(randn()) for x in generators)
```

Comment:
use a random seed for reproducibility, see e.g. https://github.com/NREL-Sienna/InfrastructureSystems.jl/pull/458

Reply:
https://github.com/NREL-Sienna/InfrastructureSystems.jl/pull/457

Reply:
I added the random seed, but we need to wait for #458 to be merged to IS first

Reply:
I am using 0 for the test as seed